modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -128,34 +117,41 @@ In this XML example, the Replay Topic Listener acts like an inbound endpoint for
 [[pushdata]]
 == Push Data to Salesforce
 
-Use as an outbound connector in your flow to push data to Salesforce. To use the connector in this capacity, simply place the Salesforce connector in your flow at any point after an inbound endpoint.
+You can use Salesforce Connector as an outbound connector in your flow to push data to Salesforce. To use the connector in this capacity, place Salesforce Connector to the right of an inbound endpoint in your flow.
 
 [[receivedata]]
 == Receive Data From Salesforce
 
-You can use the Salesforce connector as an inbound connector without wrapping the connector in a poll scope to stream data from Salesforce into your application. To use the connector in this capacity, place a Salesforce connector at the start of your flow.
+You can use Salesforce Connector as an inbound connector to stream data from Salesforce into your application. To use the connector in this capacity, place Salesforce Connector at the start of your flow and do not wrap it in a poll scope.
 
-NOTE: Studio automatically converts the connector to Salesforce (Streaming) mode. Technically, this is still the same connector, but it accesses the Salesforce Streaming API. This means that the only operation the converted connector can perform is *Subscribe Topic Listener* (that is, subscribe to PushTopic).
+Studio automatically converts the connector to Salesforce Streaming mode, which accesses the Salesforce Streaming API. The converted connector can perform only the  *Subscribe Topic Listener* operation in this mode.
 
-image::salesforce/salesforce-studio-subscribe-streaming-channel.png[subscribe streaming channel]
+*Reviewers: Is *Subscribe Topic Listener* correct in the paragraph aboe=ve, or should this be *Subscribe Channel Listener?*
```

Comment:
This is a bit outdated - meaning that in mule 4 you don't drop the connector at the start of the flow, but rather you pick one of the Sources: Subscribe Topic, Subscribe Channel, Replay Topic or Replay Channel.  - these are the "operations" that the "converted" connector can perform.